zabbix-ldap-sync

```diff
@@ -106,21 +106,24 @@ class LDAPConn(object):
                 result_dn = members[0]
                 result_attrs = members[1]
 
-            if "member" in result_attrs:
-                for member in result_attrs["member"]:
+            	if "member" in result_attrs:
+                	for member in result_attrs["member"]:
+		    		member = member.replace("(","\\28")
```

Comment:
  
I need it because our provisioning tool add bracket in LDAP path and bracket need to be replaced by their ASCII characters (rfc4515) in LDAP filters.

Envoyé depuis mon appareil mobile Samsung.

-------- Message d'origine --------
De : Marin Atanasov Nikolov <notifications@github.com> 
Date :17/10/2015  11:08  (GMT+01:00) 
À : dnaeon/zabbix-ldap-sync <zabbix-ldap-sync@noreply.github.com> 
Cc : kivengh <kevin@famillecousin.fr> 
Objet : Re: [zabbix-ldap-sync] Change sAMAcountName to userPrincipalName (#9) 

In zabbix-ldap-sync:

> @@ -106,21 +106,24 @@ class LDAPConn(object):
>                  result_dn = members[0]
>                  result_attrs = members[1]
>  
> -            if "member" in result_attrs:
> -                for member in result_attrs["member"]:
> +            	if "member" in result_attrs:
> +                	for member in result_attrs["member"]:
> +		    		member = member.replace("(","\\28")
What are the ( doing in the result and why they need to be replaced?

—
Reply to this email directly or view it on GitHub.